scripts/apps/vocabularies/controllers/VocabularyConfigController.js

```diff
@@ -1,5 +1,7 @@
 import {MEDIA_TYPES, MEDIA_TYPE_KEYS, DEFAULT_SCHEMA, VOCABULARY_SELECTION_TYPES} from '../constants';
 
+const OTHER = 'Other';
+
```

Comment:
Rename file to `.ts`. Shouldn't be much work

Reply:
done